packages/components/pages/f-account-info/src/components/AccountInfoValidationMixin.vue

```diff
@@ -0,0 +1,69 @@
+<script>
+import { validationMixin } from 'vuelidate';
+import { required, maxLength } from 'vuelidate/lib/validators';
+import { validations } from '@justeat/f-services';
+
+export default {
+    validations () {
+        const validationRules = {
+            fields: {
+                firstName: {
+                    required,
+                    maxLength: maxLength(50),
+                    isValidName: this.isValidName
+                },
+                lastName: {
+                    required,
+                    maxLength: maxLength(50),
+                    isValidName: this.isValidName
+                },
+                line1: {
+                    required
+                },
+                locality: {
+                    required
+                },
+                postcode: {
+                    required,
+                    postcodeValid: this.isValidPostcode
+                }
+            }
+        };
+
+        return validationRules;
+    },
+
+    mixins: [validationMixin],
+
+    methods: {
+        isFormInvalid () {
+            this.$v.$touch();
+            return this.$v.$invalid;
+        },
+
+        logValidationFailure () {
+            const validationDiagnostics = validations.getFormValidationState(this.$v);
+
+            this.$log.warn('Validation Failed', ['account-info', 'account-pages'], {
+                ...validationDiagnostics
+            });
+        },
+
+        isValidPostcode () {
+            return validations.isValidPostcode(this.fields.postcode, this.$i18n.locale);
+        },
+
+        isValidName (value) {
```

Comment:
Would be good to get some JSDoc comments on these methods (documenting what unicode chars this is disallowing too)